ndnfs/daemon/driver.go

```diff
@@ -80,7 +245,22 @@ func (d NdnfsDriver) Mount(r volume.MountRequest) volume.Response {
 	log.Info(DN, "Mount volume: ", r.Name)
 	d.Mutex.Lock()
 	defer d.Mutex.Unlock()
-	mnt, err := d.Client.MountVolume(r.Name)
+	var err error
+	var mnt string
+
+	nfs := fmt.Sprintf("%s:/%s", d.Config.Nedgedata, r.Name)
+	mnt = filepath.Join(d.Config.Mountpoint, r.Name)
+	log.Debug(DN, "Creating mountpoint folder: ", mnt)
+	if out, err := exec.Command("mkdir", "-p", mnt).CombinedOutput(); err != nil {
```

Comment:
Easily :)
Check playground:    https://play.golang.org/p/d9RyBKkbiN
variable 'a' is zero at the end though a new value is assigned inside if because ':=' creates a new variable with the same name - 'shadowing'